packages/extension/src/interface.ts

```diff
@@ -0,0 +1,6 @@
+import { ReadonlyPartialJSONObject } from '@lumino/coreutils';
+
+export interface ICommandBridgeRemote {
+  execute(command: string, args: ReadonlyPartialJSONObject): void;
+  listCommands(): string[];
+}
```

Comment:
Maybe we can add a few docstrings for the interface and its methods, since they will be consumed by users of the companion package?